taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -101,15 +110,16 @@ Styleguide Objects.Section
 .o-section--style-dark h6 {
   color: var(--global-color-primary--xx-light);
 }
-.o-section--style-dark + .o-section--style-dark {
-  border-top: var(--global-border-width--normal) solid
-              var(--global-color-primary--xx-light);
-}
 
-.o-section--style-light,
-.o-section--style-light::before {
+.o-section--style-light {
   color: var(--global-color-primary--dark);
   background-color: var(--global-color-primary--xx-light);
+  box-shadow: 50vw 0 var(--global-color-primary--xx-light),
+              -50vw 0 var(--global-color-primary--xx-light);
+}
+.o-section--style-light + .o-section--style-light {
+  border-top: var(--border-width) solid
+              var(--global-color-primary--normal);
 }
```

Comment:
**Explanation.**
For __light__ sections:
- Extend existing background using a `box-shadow`, not a `::before` (which had more styles later in file).
- Move the `border-top` up from below, and use variables.

Reply:
Outdated since recent comment. Resolving comment.